HISTORY.rst

```diff
@@ -56,6 +56,9 @@ Unreleased Changes
   rather than "units per area".
 * Added a ``max_pixel_fill_count`` parameter to ``routing.fill_pits`` to
   guard against pitfilling large natural depression. Defaults to 500.
+* Fixed an issue in ``align_and_resize_raster_stack`` that would ignore
+  the bounds of a feature in a mask vector if the ``"where"`` clause was
```

Comment:
Are the quotes necessary inside the ticks?

Reply:
Ah maybe it does if it's a SQL WHERE

Reply:
Right, I think it makes sense but it should really be the keyname which is `"mask_vector_where_filter"` which I changed in 70d61d2